components/app/leaderboard/LeaderboardDropdowns.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <button-emit-with-dropdown
+      button-name="Select Round"
+      class="rounded w-36 mt-10 border-white bg-white"
+      button-style="text-third"
+      label-style="-m-3 text-white"
+      :menu-items="roundItems"
+      @selected="setItems"
+    />
+  </div>
+</template>
+
+<script>
+import ButtonEmitWithDropdown from '@/components/common/ButtonEmitWithDropdown.vue';
+
+export default {
+  name: 'LeaderBoardDropdowns',
```

Comment:
nitpick(blocking): we used **Leaderboard** before, Keep the previous naming style